chain/x/oracle/keeper/querier.go

```diff
@@ -95,10 +95,10 @@ func queryRequestByID(ctx sdk.Context, path []string, k Keeper) ([]byte, error)
 		return nil, err
 	}
 	reports := k.GetReports(ctx, types.RequestID(id))
-	result, _ := k.GetResult(ctx, types.RequestID(id))
+	result := k.MustGetResult(ctx, types.RequestID(id))
```

Comment:
I think it should use HasResult first if the result doesn't exist it should return nil not panic endpoint.